left_to_do.txt

```diff
@@ -1 +1,3 @@
 - Add notification to notify participants of winners
+- Check if teams can submit projects still if status is judging, finished, deleted
```

Comment:
Teams should only be able to submit projects until a deadline.... not if the status is judging, finished, or deleted.... that would royally mess up the scoring, no? Ideally, like deployment, get the project submitted initially, and can edit where needed up until the deadline to submit... no late submissions, once it's changed over to "Judging", done, no more submissions permitted (ie: you snooze, you lose).

Reply:
Okay, sure. Just was not sure we wanted to be that strict :smile:

Reply:
I did add this in to only allow teams to submit in status `hack_in_progress`.